tests/functional/adapter/test_query_comment.py

```diff
@@ -6,6 +6,8 @@
     BaseNullQueryComments,
     BaseEmptyQueryComments,
 )
+from dbt.adapters.redshift.__version__ import version
```

Comment:
Temp workaround, need to fix in test package: https://github.com/dbt-labs/dbt-adapters/issues/56